lib/logger.js

```diff
@@ -0,0 +1,55 @@
+const debugLib = require('debug')
+const errorLogger = debugLib('electronpdf:error:')
+const infoLogger = debugLib('electronpdf:info:')
+const debugLogger = debugLib('electronpdf:debug:')
+
+const LEVELS = {
```

Comment:
Unless I'm not understanding something, this logger object looks like it could be simplified.  Why not just store your loggers like this?

```
const loggers = {
    info: infoLogger,
    debug: debugLogger,
    error: errorLogger
}
```